.github/workflows/docker.yml

```diff
@@ -28,5 +28,5 @@ jobs:
           file: ./Dockerfile
           platforms: linux/amd64,linux/arm,linux/arm64/v8
           push: true
-          tags: tzahi12345/youtubedl-material:latest
-  
\ No newline at end of file
+          tags: tzahi12345/youtubedl-material:nightly
+  
```

Comment:
```suggestion
```

I think there are still two final new lines.

Reply:
Yeah I think GitHub warns if it's EOF without a new line